src/mappings/horizonStaking.ts

```diff
@@ -270,4 +262,14 @@ export function handleThawRequestFulfilled(event: ThawRequestFulfilled): void {
     request.tokens = event.params.tokens
     request.valid = event.params.valid
     request.save()
+}
+
+export function handleTokensToDelegationPoolAdded(event: TokensToDelegationPoolAdded): void {
+    let indexer = Indexer.load(event.params.serviceProvider.toHexString())!
+    let provision = createOrLoadProvision(event.params.serviceProvider, event.params.verifier, event.block.timestamp)
+    provision.delegatedTokens = provision.delegatedTokens.plus(event.params.tokens)
```

Comment:
We likely also want to update `provision.delegationExchangeRate` here no? We are not issuing new shares but the tokens backing those shares change.

Reply:
I'm not sure but I think I updated this when I worked on the delegation code (as I worked on this a long time before getting to the full delegation support PR), will double check and fix if missing